.github/workflows/deploy.yml

```diff
@@ -0,0 +1,41 @@
+name: Deploy
+
+on:
+  push:
+    tags:
+      - 'preprod'
+      - 'production'
```

Comment:
Out of curiosity, do we already need the `production` tag?

Reply:
We can put it in the production PR if you want.